index.html

```diff
@@ -3191,13 +3176,12 @@ <h3>Disputes</h3>
     },
   }</span>,
   "issuer": "https://example.com/people#me",
-  "issuanceDate": "2017-12-05T14:27:42Z",
-  "proof": { <span class="comment">...</span> }
+  "issuanceDate": "2017-12-05T14:27:42Z"
 }
         </pre>
 
         <p>
-In the above <a>verifiable credential</a> the <a>issuer</a> is claiming that
+In the above <a>credential</a> the <a>issuer</a> is claiming that
```

Comment:
```suggestion
In the above <a>credential</a>, the <a>issuer</a> is claiming that
```

Reply:
Ok accepted

Reply:
Note that there are buttons on each suggested change, to "Commit suggestion" or "Add suggestion to batch", the latter of which leads to a subsequent "Commit batch", which avoid the need for "Ok accepted" comments and *should* make applying such suggestions faster and easier.

Reply:
Because I am a git novice, I found whilst updating my previous PR (808) that committing changes on the web site messed up my local GitHub copy and I could no longer push a revised version from my PC to the web, as they had become out of sync. So I prefer to make all the changes on my local GitHub copy then push the revised version up to web.

Reply:
I know there's a way around that issue. I can't tell you how to access it because I don't use local `git` at all; all my interactions are through GitHub's web interface. :-/

Reply:
If you like how everything reads on your local branch the simple answer is to force push to the remote branch. This can be done with something like the following:

`git push --force origin Removing-Proof`

This is effectively overwriting any changes that exists on github to match the git log on your local computer.

Reply:
If we could spend a few minutes at the next VCWG meeting to discuss how best to author PRs for novices I would appreciate it (web vs desktop vc command line etc.) Ease of use being the primary factor :-)

Reply:
> [@David-Chadwick] If we could spend a few minutes at the next VCWG meeting to discuss how best to author PRs for novices I would appreciate it (web vs desktop vc command line etc.) Ease of use being the primary factor :-)

Best to reply to the meeting's AGENDA email, or email the chairs & editors directly, with this request.

(My personal answer is to use the GitHub interface as much as possible, as that seems to have the fewest foot-guns, and really is set up pretty well for non-coders and coders alike. I have only rarely had to appeal to someone else to use the command-line tools to achieve something I couldn't do easily though the website, and those times were mostly about global search-and-replace of a few typos that had permeated a repo.)

Reply:
If we're unable to get to this, I'm happy to find a time that we could cover some of the basics in a tutorial that we post in the CCG as a recording since I think it's useful for some of the initiatives they're doing to be more inclusive as well.